.github/workflows/scorecard.yaml

```diff
@@ -17,7 +17,7 @@ permissions: read-all
 jobs:
   analysis:
     name: Scorecards analysis
-    runs-on: ubuntu-22.04
+    runs-on: github-arm64-2c-8gb
```

Comment:
Using this tiny lil guy because why not. It's a small task, and this runner is good enough.